pysisyphus/default.nix

```
@@ -0,0 +1,137 @@
+{ fetchPypi, buildPythonPackage, lib, writeTextFile, writeScript, makeWrapper,
+ # Python dependencies
+ autograd, dask, distributed, h5py, jinja2, matplotlib, numpy, natsort, pytest, pyyaml, rmsd, scipy,
+ sympy, scikitlearn, qcengine, ase,
+ # Runtime dependencies
+ jmol, multiwfn, xtb, openmolcas, pyscf, psi4, wfoverlap, nwchem, orca ? null , turbomole ? null,
+ gaussian ? null, gamess-us ? null, cfour ? null, molpro ? null
+}:
+let
+  psi4Wrapper = writeScript "psi4.sh" ''
+    ${psi4}/bin/psi4 -o stdout $1
+  '';
+  pysisrc =
+    let
+      gaussian16Conf = {
+        cmd = "${gaussian}/bin/g16";
+        formchk_cmd = "${gaussian}/bin/formchk";
+        unfchk_cmd = "${gaussian}/bin/unfchk";
+      };
+      openmolcasConf = {
+        cmd = "${openmolcas}/bin/pymolcas";
+      };
+      orcaConf = {
+        cmd = "${orca}/bin/orca";
+      };
+      psi4Conf = {
+        cmd = "${toString psi4Wrapper}";
```

This should be sufficient.
```suggestion
        cmd = "${psi4Wrapper}";
```